client/client.go

```diff
@@ -33,7 +33,10 @@ var (
 	errTooManyRedirectChecks = errors.New("client: too many redirect checks")
 )
 
-var DefaultRequestTimeout = 5 * time.Second
+var (
+	DefaultRequestTimeout = 5 * time.Second
+	DefaultSyncInterval   = 10 * time.Second
```

Comment:
I think we should doc that rather than put it into our code. No one will find out it without documentation. If you have the documentation, then you do not need this at all.